packages/ag-charts-enterprise/src/features/context-menu/contextMenu.ts

```diff
@@ -285,23 +267,27 @@ export class ContextMenu extends _ModuleSupport.BaseModuleInstance implements _M
         return el;
     }
 
-    private createActionElement({ id, label, action }: ContextMenuAction): HTMLElement {
+    private createActionElement(pickedNode: ContextPickedNode, { id, label, action }: ContextMenuAction): HTMLElement {
         if (id && ContextMenu.disabledActions.has(id)) {
             return this.createDisabledElement(label);
         }
-        return this.createButtonElement(label, action);
+        return this.createButtonElement(pickedNode, label, action);
     }
 
-    private createButtonElement(label: string, callback: (params: ContextMenuActionParams) => void): HTMLElement {
+    private createButtonElement(
+        pickedNode: ContextPickedNode,
```

Comment:
So down to here should be `pickedNode?: ContextPickedNode`

Reply:
Done. This change was reverted.